gnuradio-default.lwr

```diff
@@ -36,3 +36,5 @@ config:
       gitbranch: maint-3.8
     gr-lfast:
       gitbranch: maint-3.8
+    gr-iqbal:
+      gitbranch: gr3.8
```

Comment:
This fixes https://github.com/gnuradio/gr-recipes/issues/246 for me; I was going to submit a PR but this will probably do.